lib/setopt.h

```diff
@@ -23,6 +23,7 @@
  ***************************************************************************/
 
 CURLcode Curl_setstropt(char **charp, const char *s);
+CURLcode Curl_setblobopt(char **charp, const char *sblob);
```

Comment:
Why doesn't it store a `curl_blob` struct?

Reply:
see comment on Curl_setblobopt . I do it